ui/app/AppLayouts/Profile/views/WalletView.qml

```diff
@@ -48,6 +44,12 @@ ScrollView {
 
             MainView {
                 walletStore: root.walletStore
+                anchors.topMargin: 64
```

Comment:
use the one in Theme.qml or we can also add a new one

Reply:
done